extensions/amp-tiktok/0.1/test/validator-amp-tiktok.out

```diff
@@ -37,13 +37,9 @@ FAIL
 |  <body>
 |    <!-- Valid -->
 |    <amp-tiktok width="500" height="500" data-src="6943753342808034566">
->>   ^~~~~~~~~
-amp-tiktok/0.1/test/validator-amp-tiktok.html:38:2 Tag 'amp-tiktok' must have a minimum of 1 child tags - saw 0 child tags. (see https://amp.dev/documentation/components/amp-tiktok)
 |    </amp-tiktok>
 |    <!-- Valid -->
 |    <amp-tiktok
->>   ^~~~~~~~~
-amp-tiktok/0.1/test/validator-amp-tiktok.html:41:2 Tag 'amp-tiktok' must have a minimum of 1 child tags - saw 0 child tags. (see https://amp.dev/documentation/components/amp-tiktok)
 |      width="700"
 |      height="800"
 |      data-src="https://www.tiktok.com/@scout2015/video/6943753342808034566">
```

Comment:
This URL doesn't match your regexp, which is why the test doesn't pass. You're removing the error in your test, but it is still an error with the current rules. If I'm reading it correctly (something that I never know with regexps), it only allows numbers after the first slash in the path, however this example includes other stuff like letters.

Reply:
It does match actually, as the pattern is `(https://www\.tiktok\.com/.*)?\\d+.*` meaning it allows the value to end in digits followed by any number of other characters (`.*`). 

When I ran `python3 build.py --update_tests` locally it removed this.

Reply:
Hm, apologies for the bad advice here. I didn't notice the `.*` before the `\\d`. Regexps are hard.